### DISCUSSION TITLE
Submit_feed

### DISCUSSION THREAD

**Discussion by Nileshca** [amzn/selling-partner-api-models#3536] [General]
When I am using the submit_feed I am getting a response:
 try:
        response = api.submit_feed(feed_type='POST_ORDER_FULFILLMENT_DATA', file=xml_io, marketplace_ids=["ATVPDKIKX0DER"], content_type='text/xml')
        err = response.errors
        print(err)
        feed_id = response.feed_submission_info.feed_submission_id
    except Exception as e:
        error_message = str(e)
        print(f"Feed submission failed. Error message: {error_message}")
    return feed_id
(<sp_api.base.ApiResponse.ApiResponse object at 0x0000027D47B736D0>, <sp_api.base.ApiResponse.ApiResponse object at 0x0000027D4909A920>)
Any idea what is this response? I was expecting feed_id

**Comment by Nileshca** [comment]
I figured out the answer.

**Reply by sungolivia** [reply]
hey @Nileshca can you share as others might run into same issue as you? Thanks!
